integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOpenshiftIstioMiiDomain.java

```diff
@@ -0,0 +1,382 @@
+  private String configMapName = "dynamicupdate-istio-configmap";
+  private final String clusterName = "cluster-1"; // do not modify
+  private final String adminServerPodName = domainUid + "-admin-server";
+  private final String managedServerPrefix = domainUid + "-managed-server";
+  private final String workManagerName = "newWM";
+  private final int replicaCount = 2;
+
+  private static String testWebAppWarLoc = null;
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher
+  */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a new unique opNamespace
+    logger.info("Assign unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Assign unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+    
+    // edit service member roll to include operator and domain namespace so that 
+    // Openshift service mesh can add istio side cars to the operator and WebLogic pods.
+    Path smrYaml = Paths.get(WORK_DIR, "openshift", "servicememberroll.yaml");
+    assertDoesNotThrow(() -> {
+      FileUtils.copy(Paths.get(RESOURCE_DIR, "openshift", "servicememberroll.yaml"),
+          smrYaml);
+      FileUtils.replaceStringInFile(smrYaml.toString(), "OPERATOR_NAMESPACE", opNamespace);
+      FileUtils.replaceStringInFile(smrYaml.toString(), "DOMAIN_NAMESPACE", domainNamespace);
+    });
+    logger.info("Run kubectl to create the service member roll");
+    CommandParams params = new CommandParams().defaults();
+    params.command("kubectl apply -f " + smrYaml.toString());
+    boolean result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create service member roll");
+
+    // install and verify operator with istio side car injection set to true
+    String opSa = opNamespace + "-sa";
+    HelmParams opHelmParams
+        = new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+            .namespace(opNamespace)
+            .chartDir(OPERATOR_CHART_DIR);
+    
+    installAndVerifyOperator(opNamespace,
+        opSa, // operator service account
+        false, // with REST api enabled
+        0, // externalRestHttpPort
+        opHelmParams, // operator helm parameters
+        false, // ElkintegrationEnabled
+        null, // domainspaceSelectionStrategy
+        null, // domainspaceSelector
+        false, // enableClusterRolebinding
+        "INFO", // operator pod log level
+        -1, // domainPresenceFailureRetryMaxCount
+        -1, // domainPresenceFailureRetrySeconds
+        true, // openshift istio injection
+        domainNamespace // domainNamespace
+    );
+  }
+
+  /**
+   * Create a domain using model-in-image model.
```

Comment:
Where we do set the 
      (a) localhostBindingsEnabled to true in domain/istio configuration.
      (b) annotate  the server pod with sidecar.istio.io/inject: "true"

Reply:
inside the test method after creating the domain resource

Reply:
inside test method after creating the domain resource these two parameters are set